spotifycli/spotifycli.py

```diff
@@ -150,7 +151,9 @@ def show_version():
 
 def get_song():
     metadata = get_spotify_property("Metadata")
-    artist = ", ".join(metadata['xesam:artist'][1])
+    artist = metadata['xesam:artist'][1]
+    if isinstance(artist, list):
+        artist = ", ".join(artist)
```

Comment:
OK, so this change is an artifact of dbus message mocking.  I'll share more detail in a following comment here, but as it is right now, the `get_song` function can take artist as a string or list of strings.  It shouldn't have _any_ effect on the operation of the library.